homeassistant/components/apcupsd/config_flow.py

```diff
@@ -52,9 +52,8 @@ async def async_step_user(
 
         # Test the connection to the host and get the current status for serial number.
         coordinator = APCUPSdCoordinator(self.hass, host, port)
```

Comment:
Usually we don't create coordinators in config flows.

In a future PR it would be nice to replace this with `await aioapcaccess.request_status(self._host, self._port)`

Reply:
Sounds good, will do!